en/docs/how-tos/support-custom-claims-mapping.md

```diff
@@ -0,0 +1,99 @@
+     #support custom claim mapping
+     claimMapperClassName = "org.wso2.micro.gateway.jwtTransformer.DefaultJwtTransformer"
+     [[jwtTokenConfig.claims]]
+       remoteClaim = "scp"
+       localClaim = "scope" 
+```
+         
+      
+2. If the format of the remote claim value differs from the custom claims then you should write a class to transform it into the correct format and define the claim mapping class name in the config file.
+
+     The following subsections explain how you can work with claims value transformation. Here the existing scope value is in an array and it shoud be change to string with space.
+#### Writing JWT transformer for claims value
+Microgateway defines java interfaces to write the JWT value transformer. The interface for the transformer defined below. Developers can implement these interfaces to achieve custom transformation logic. 
+```` java
+package org.wso2.micro.gateway.jwt.transformer;
+ 
+import java.util.Map;
+ 
+/**
+ * Defines the interface for writing the  jwt tranformation.
+ */
+public interface JWTValueTransformer {
+ 
+    /**
+     *  Transform the custom claims  into required format.
+     * @param jwtClaims jwtClaims from given JWT
+     * @return transformed JWT Claims
+     */
+    Map<String, Object> transformJWT(Map<String, Object> jwtClaims);
+}
+````
+####  Writing a JWT claims value transformer
+``` java
+package org.mgw.jwt.jwtvaluetransformer;
+ 
+import org.wso2.micro.gateway.jwt.transformer.JWTValueTransformer;
+ 
+import java.util.Map;
+ 
+/**
+ * This class is for default Jwt transformer.
+ */
+public class customJwtTransformer implements JWTValueTransformer {
+ 
+    @Override
+    public Map<String, Object> transformJWT(Map<String, Object> jwtClaims) {
+        String scope = "";
+        if (jwtClaims.containsKey("scope")) {
+            if (jwtClaims.get("scope") instanceof Object[]) {
+                for (int i = 0; i < ((Object[]) jwtClaims.get("scope")).length; i++) {
+                    scope += ((Object[]) jwtClaims.get("scope"))[i] + " ";
+                }
+                scope = scope.trim();
+            }
+            jwtClaims.put("scope", scope);
+        }
+        return jwtClaims;
+    }
+}
+```
+
+    !!! info
+        Add the microgateway JWT transformer dependency.
+        The java JWT transformer project will require the following dependency
+        **JWT transformer dependency**
+        ``` java
+        <dependency>
+            <groupId>org.wso2.am.microgw</groupId>
+            <artifactId>mgw-jwt-transformer</artifactId>
+            <version>3.2.0</version>
+        </dependency>
+        ```
+#### Adding JWT transformer to the project 
+Once JWT claims value transformer is written, then the JWT  transformer project should be built and the output jar should be placed in the <MGW-project>/lib directory. If third-party libraries are used when writing jwt claims mapping transformer, these custom jars should also be placed in the same directory.
```

Comment:
Once JWT claims value transformer is written, then the JWT  transformer project should be built and the output jar should be placed in the <MGW-project>/lib directory. If third-party libraries are used when writing **JWT** claims mapping transformer, these custom jars should also be placed in the same directory.